docs/source/delta-sharing.md

```diff
@@ -0,0 +1,204 @@
+
+  spark.read.format("delta").option("versionAsOf", version).load(tablePath)
+  ```
+  ```scala
+  spark.read.format("delta").option("timestampAsOf", timestamp_string).load(tablePath)
+
+  spark.read.format("delta").option("versionAsOf", version).load(tablePath)
+  ```
+
+The `timestamp_expression` and `version` share the same syntax as [delta](delta-batch.md#timestamp-and-version-syntax).
+
+## Read CDF
+
+Once the provider turns on CDF on the original delta table and shares it with history through Delta Sharing, the recipient can query CDF of a Delta Sharing table similar to CDF of a delta table.
+
+TODO: check timestamp format.
+
+.. code-language-tabs::
+
+  ```sql
+  CREATE TABLE mytable USING deltaSharing LOCATION '<profile-file-path>#<share-name>.<schema-name>.<table-name>';
+  
+  -- version as ints or longs e.g. changes from version 0 to 10
+  SELECT * FROM table_changes('mytable', 0, 10)
+  
+  -- timestamp as string formatted timestamps
+  SELECT * FROM table_changes('mytable', '2021-04-21 05:45:46', '2021-05-21 12:00:00')
+  
+  -- providing only the startingVersion/timestamp
+  SELECT * FROM table_changes('mytable', 0)
+  ```
+  ```python
+  table_path = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+  
+  # version as ints or longs
+  spark.read.format("deltaSharing") \
+    .option("readChangeFeed", "true") \
+    .option("startingVersion", 0) \
+    .option("endingVersion", 10) \
+    .load(tablePath)
+  
+  # timestamps as formatted timestamp
+  spark.read.format("deltaSharing") \
+    .option("readChangeFeed", "true") \
+    .option("startingTimestamp", '2021-04-21 05:45:46') \
+    .option("endingTimestamp", '2021-05-21 12:00:00') \
+    .load(tablePath)
+  
+  # providing only the startingVersion/timestamp
+  spark.read.format("deltaSharing") \
+    .option("readChangeFeed", "true") \
+    .option("startingVersion", 0) \
+    .load(tablePath)
+  ```
+
+  ```scala
+  val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+  // version as ints or longs
+  spark.read.format("deltaSharing")
+    .option("readChangeFeed", "true")
+    .option("startingVersion", 0)
+    .option("endingVersion", 10)
+    .load(tablePath)
+
+  // timestamps as formatted timestamp
+  spark.read.format("deltaSharing")
+    .option("readChangeFeed", "true")
+    .option("startingTimestamp", "2024-01-18 05:45:46")
+    .option("endingTimestamp", "2024-01-18 12:00:00")
+    .load(tablePath)
+  
+  // providing only the startingVersion/timestamp
+  spark.read.format("deltaSharing")
+    .option("readChangeFeed", "true")
+    .option("startingVersion", 0)
+    .load(tablePath)
+  ```
+
+## Streaming
+
+Delta Sharing Streaming is deeply integrated with [Spark Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) through `readStream`, 
+and able to connect with any sink that is able to perform `writeStream`.
```

Comment:
nit: remove the new line.

Reply:
I intended to make them separate as it's a new topic.